public/components/marketing-forms/5.html

```diff
@@ -0,0 +1,104 @@
+<!--
+  Heads up! 👋
+
+  Plugins:
+    - @tailwindcss/forms
+-->
+
+<div class="flex justify-center p-6 font-sans antialiased">
+  <section class="w-full max-w-md space-y-8 divide-y-2 bg-white md:max-w-4xl">
```

Comment:
Don't really think we need `divide-y-2` if it's only used in one place, might be better to just specifiy `border-t-2` where it is needed.

Reply:
Indeed, I'll use the `border-t` utility.